.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+version: 2.1
+executors:
+    texture_exec:
+        macos:
+            xcode: 10.1.0
+        # working_directory: ~/project
+commands:
+    buildsh:
+        parameters:
+            mode: {type: string}
+        steps:
+            # - {attach_workspace: {at: ~/project}}
+            - checkout
+            - {run: {name: 'Fetch cocoapods spec repo from cache', command: "curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf\n"}}
+            - {run: xcrun simctl list}
```

Comment:
Huh why?

Reply:
If you don't, the simulator engine sometimes barfs when it comes time to run